src/ride_type.cpp

```diff
@@ -226,6 +226,23 @@ const RideType *RideInstance::GetRideType() const
  * Immediately remove this ride from all voxels it currently occupies.
  */
 
+/**
+ * \fn void RideInstance::InsertIntoWorld()
+ * Ensure that this shop is linked into the voxels it is meant to occupy.
+ */
+
+/**
+ * \fn void ShouldDrawPiece::InsertIntoWorld()
+ * Whether the specified piece of the ride should be visible to the player.
+ * @param voxel_number The instance data of the piece to check.
+ * @return `false` if the ride piece is meant to be fully invisible; `true` otherwise.
```

Comment:
I typically write that as a statement like you wrote at line 236. Iff the answer to the statement is true you must return `true`.